docs/pages/api-reference/operators.md

```diff
@@ -101,6 +101,19 @@ The `aggregate` operator has the following parameters:
 
 <pre snippet="api-reference/operators_ref#aggregate"></pre>
 
+### Groupby/ First
+
+Fennel allows you to groupby and get the first entry (by timestamp) in each group using the `first` operator.
+The api requires you to chain the `groupby` operator with the `first` operator.
+
+The `groupby` operator takes the following parameters:
+
+1. `*args: str` - positional arguments that specify the list of fields to group by.
```

Comment:
Not sure if this is needed here? Having this in two places makes it harder to keep the documentation upto date as the groupby operator evolves.

Reply:
Fyi we dont have a seperate groupby operator documentation, it is combined with aggregate

Reply:
+1 to what Nikhil is saying (about having documentaiton for groupby duplicated), but also `groupby in itself is not a full operator (e.g. we can't end a pipeline at `groupby`) which makes it challenging. Any ideas for this @nikhilgarg28 ?